common/mqtt_client/Cargo.toml

```diff
@@ -12,6 +12,7 @@ thiserror = "1.0"
 tokio = { version = "1.8", features = ["sync", "macros"] }
 mockall = "0.10"
 async-trait = "0.1"
+anyhow = "1.0"
```

Comment:
I can't see this being used in this PR... Is this intentional?

Reply:
fixed